src/Microsoft.Identity.Web.OWIN/OwinTokenAcquisitionHost.cs

```diff
@@ -50,22 +50,13 @@ public MergedOptions GetOptions(string? authenticationScheme, out string effecti
         {
             effectiveAuthenticationScheme = GetEffectiveAuthenticationScheme(authenticationScheme);
             var mergedOptions = _mergedOptionsMonitor.Get(effectiveAuthenticationScheme);
-
             if (!mergedOptions.MergedWithCca)
             {
                 _ccaOptionsMonitor.Get(effectiveAuthenticationScheme);
             }
 
-            MergedOptions.UpdateMergedOptionsFromMicrosoftIdentityOptions(_microsoftIdentityOptionsMonitor.Get(effectiveAuthenticationScheme), mergedOptions);
-
-            if (string.IsNullOrEmpty(mergedOptions.Instance) || string.IsNullOrEmpty(mergedOptions.ClientId))
-            {
-                MicrosoftAuthenticationOptions? microsoftAuthenticationOptions = _microsoftAuthenticationOptionsMonitor.Get(effectiveAuthenticationScheme);
-                if (microsoftAuthenticationOptions != null)
-                {
-                    MergedOptions.UpdateMergedOptionsFromMicrosoftAuthenticationOptions(microsoftAuthenticationOptions, mergedOptions);
-                }
-            }
+            _microsoftIdentityOptionsMonitor.Get(effectiveAuthenticationScheme);
```

Comment:
We might want comments to explain that this is to force the PostConfigure

Reply:
fixed.